tests/src/Context/LogicalThreadContextTest.cs

```diff
@@ -17,7 +17,7 @@
 //
 #endregion
 
-#if NET_4_5 || NETSTANDARD1_3
+#if NET_4_5 || NETCOREAPP1_0 || NETCOREAPP2_0
```

Comment:
I find that less clear because it confuses .NET Standard and .NET Core, and suggests targeting `netstandard1.3` and `netcoreapp1.0` simultaneously, which is bizarre. A .NET Standard version is similar to a PCL profile here: an abstract, runtime-agnostic API contract, not a platform on which code runs. Test projects build executables because tests must run on some platform, like .NET Core.